docs/getting_started.rst

```diff
@@ -1,4 +1,4 @@
-Getting Started
+Getting Started 🦥
 ===============
```

Comment:
```suggestion
Getting Started 🦥
==================
```
...since the title underline has to be the same length as the title, and emojis apparently count as two spaces.